components/fxa-client/src/http_client.rs

```diff
@@ -344,6 +353,43 @@ impl FxAClient for Client {
         Ok(())
     }
 
+    fn attached_clients(&self, config: &Config, refresh_token: &str) -> Result<Vec<GetAttachedClientResponse>> {
+        let url = config.auth_url_path("v1/account/attached_clients")?;
+        let request =
+            Request::get(url).header(header_names::AUTHORIZATION, bearer_token(refresh_token))?;
+        let mut attached_clients: Vec<GetAttachedClientResponse> = Self::make_request(request)?.json()?;
+
+        for mut attached_client in &mut attached_clients {
```

Comment:
If possible I'd make that kind of post-processing outside of the `http_client` module

Reply:
@eoger I moved _most_ of the post-processing logic to `util.rs` but not all because I didn't want to create circular dependencies between `util` and `http_client`. If this isn't what you had in mind, please let me know. Not sure if there's a better place to put all of this.

Reply:
Thanks! Sorry to be annoying, but could we also move the whole loop in the caller?

Reply:
Thinking about this more, how about we let go of all that post-processing code and let the caller decide what it wants to do with the `last_access_time` time. It seems wrong to me to make assumptions here.

Reply:
Yeah, I'll do that. I wasn't completely convinced that mimicking the logic that's [in desktop](https://searchfox.org/mozilla-central/source/services/fxaccounts/FxAccounts.jsm#464) was worthwhile.